Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/BoundsControl/Scripts/BoundsControlRuntimeExample.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using System.Text;
+using TMPro;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Experimental.Demos
+{
+    /// <summary>
+    /// TODO: This demo probably needs to be adjusted
+    /// Currently it's just a copy of BoundingBoxExampleTest
+    /// </summary>
+    public class BoundsControlRuntimeExample : MonoBehaviour, IMixedRealitySpeechHandler
+    {
+        public TextMeshPro statusText;
```

Comment:
Mentioning the [encapsulation guideline](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation) again here, though this class is marked as a TODO, so it could be addressed at the same time as that.